tests/vmi_lifecycle_test.go

```diff
@@ -1512,7 +1512,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			}, 30*time.Second, 2*time.Second).Should(Equal(expectedStatus))
 		}
 
-		It("[test_id:7479] should succeed", func() {
+		It("[test_id:7479] should succeed", decorators.Conformance, func() {
```

Comment:
7480 seem a good candidate and this test seems a subset of it

Reply:
Done